modules/developers-guide/pages/tutorials/define-an-actor.adoc

```diff
@@ -239,9 +239,9 @@ Note that if you are running the Internet Computer network in a separate termina
 
 == Stop the local network
 
-After you finish experimenting with your program, you can stop the local Internet Computer network so that it doesn’t continue running in the background.
+After you finish experimenting with your program, you can stop the local Internet Computer so that it doesn’t continue running in the background.
```

Comment:
```suggestion
After you finish experimenting with your program, you can stop the local Internet Computer so that it doesn’t continue running in the background.
```
```suggestion
After you finish experimenting with your program, you can stop the local canister execution environment so that it doesn’t continue running in the background.
```